utils/kubernetes/kube_state_processor.py

```diff
@@ -120,6 +167,43 @@ def kube_node_status_allocatable_pods(self, message, **kwargs):
             tags = ['node:{}'.format(self._extract_label_value("node", metric.label))]
             self.gauge(metric_name, val, tags)
 
+    # Labels attached: daemonset, namespace
+    def kube_daemonset_status_current_number_scheduled(self, message, **kwargs):
+        """ Number of nodes running at least 1 daemon pod that are supposed to run one. """
```

Comment:
It might be easier to parse, but is it English?

The number of nodes running at least one daemon pod and that are supposed to?
The number of nodes running at least one daemon pod which are supposed to?
The number of nodes that are running at least one daemon pod and are supposed to?

I think the last one might be the best.